docs/working-with-projects/project-configuration/configure-webhooks.md

```diff
@@ -94,4 +133,4 @@ As a Harbor system administrator, you can enable and disable webhook notificatio
 1. Go to **Configuration** > **System Settings**.
 1. Scroll down and check or uncheck the **Webhooks enabled** check box.
 
-    ![Enable/disable webhooks](../../../img/webhooks4.png)
+    ![Enable/disable webhooks](../../img/webhooks4.png)
```

Comment:
'../../../' is correct, this is a behavior of the system that hooks up to the website, basically the img is in its own directory, this needs to be changed back or image won't show properly on website